linkcheck/checker/httpurl.py

```diff
@@ -315,7 +303,8 @@ def check_response(self):
 
             if self.url_connection.status_code == 429:
                 self.add_warning(
-                    "Rate limited (Retry-After: %s)" % self.getheader(_("Retry-After")),
+                    "Rate limited (Retry-After: %s)"
+                    % self.headers.get(_("Retry-After")),
```

Comment:
Wait, are we translating the name of an HTTP header????  That would be a bug (the fix for which belongs to a separate PR).

Reply:
Raised issue #449.